packages/twenty-server/src/engine/metadata-modules/index-metadata/index-metadata.entity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  JoinColumn,
+  ManyToOne,
+  Relation,
+  OneToMany,
+} from 'typeorm';
+
+import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
+import { IndexFieldMetadataEntity } from 'src/engine/metadata-modules/index-field-metadata/index-field-metadata.entity';
+
+@Entity('indexMetadata')
+export class IndexMetadataEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ nullable: false })
+  name: string;
+
+  @Column({ nullable: true })
+  expression: string;
```

Comment:
I would remove it for now, I don't feel like it's more complex to add it later than now so it can be a bit misleading to have it but not implemented

Reply:
Sure, you are right 👍